Database Structures/DATAFILES/LightMeanValues.json

```diff
@@ -0,0 +1,111 @@
+
+{
+	"NORTH": [
+		{
+			"DEG_N" : 60,
+			"MONTH_VALS" : [0.15,0.20,0.26,0.32,0.38,0.41,0.40,0.34,0.28,0.22,0.17,0.13]
+		},
+		{
+			"DEG_N" : 55,
+			"MONTH_VALS" : [0.17,0.21,0.26,0.32,0.36,0.39,0.38,0.33,0.28,0.23,0.18,0.16]
+		},
+		{
+			"DEG_N" : 50,
+			"MONTH_VALS" : [0.19,0.23,0.27,0.31,0.34,0.36,0.35,0.32,0.28,0.24,0.20,0.18]
+		},
+		{
+			"DEG_N" : 45,
+			"MONTH_VALS" : [0.20,0.23,0.27,0.30,0.34,0.35,0.34,0.32,0.28,0.24,0.21,0.20]
+		},
+		{
+			"DEG_N" : 40,
+			"MONTH_VALS" : [0.22,0.24,0.27,0.30,0.32,0.34,0.33,0.31,0.28,0.25,0.22,0.21]
+		},
+		{
+			"DEG_N" : 35,
+			"MONTH_VALS" : [0.23,0.25,0.27,0.29,0.31,0.32,0.32,0.30,0.28,0.25,0.23,0.22]
+		},
+		{
+			"DEG_N" : 30,
+			"MONTH_VALS" : [0.24,0.25,0.27,0.29,0.31,0.32,0.31,0.30,0.28,0.26,0.24,0.23]	
+		},
+		{
+			"DEG_N" : 25,
+			"MONTH_VALS" : [0.24,0.26,0.27,0.29,0.30,0.31,0.31,0.29,0.28,0.26,0.25,0.24]
+		},
+		{
+			"DEG_N" : 20,
+			"MONTH_VALS" : [0.25,0.26,0.27,0.28,0.29,0.30,0.30,0.29,0.28,0.26,0.25,0.25]
+		},
+		{
+			"DEG_N" : 15,
+			"MONTH_VALS": [0.26,0.26,0.27,0.28,0.29,0.29,0.29,0.28,0.28,0.27,0.26,0.25]
+		},
+		{
+			"DEG_N" : 10,
+			"MONTH_VALS" : [0.26,0.27,0.27,0.28,0.28,0.29,0.29,0.28,0.28,0.27,0.26,0.26]
+		},
+		{
+			"DEG_N" : 5,
+			"MONTH_VALS" : [0.27,0.27,0.27,0.28,0.28,0.28,0.28,0.28,0.28,0.27,0.27,0.27]
+		},
+		{
+			"DEG_N" : 0,
+			"MONTH_VALS" : [0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27]
+		}
+	],
+	"SOUTH" :[
+		{
+			"DEG_S" : 60,
+			"MONTH_VALS" : [0.40,0.34,0.28,0.22,0.17,0.13,0.15,0.20,0.26,0.32,0.38,0.41]
+		},
+		{
+			"DEG_S" : 55,
+			"MONTH_VALS" : [0.38,0.33,0.28,0.23,0.18,0.16,0.17,0.21,0.26,0.32,0.36,0.39]
+		},
+		{
+			"DEG_S" : 50,
+			"MONTH_VALS" : [0.35,0.32,0.28,0.24,0.20,0.18,0.19,0.23,0.27,0.31,0.34,0.36]
+		},
+		{
+			"DEG_S" : 45,
+			"MONTH_VALS" : [0.34,0.32,0.28,0.24,0.21,0.20,0.20,0.23,0.27,0.30,0.34,0.35]
+		},
+		{
+			"DEG_S" : 40,
+			"MONTH_VALS" : [0.33,0.31,0.28,0.25,0.22,0.21,0.22,0.24,0.27,0.30,0.32,0.34]
+		},
+		{
+			"DEG_S" : 35,
+			"MONTH_VALS" : [0.32,0.30,0.28,0.25,0.23,0.22,0.23,0.25,0.27,0.29,0.31,0.32]
+		},
+		{
+			"DEG_S" : 30,
+			"MONTH_VALS" : [0.31,0.30,0.28,0.26,0.24,0.23,0.24,0.25,0.27,0.29,0.31,0.32]	
+		},
+		{
+			"DEG_S" : 25,
+			"MONTH_VALS" : [0.31,0.29,0.28,0.26,0.25,0.24,0.24,0.26,0.27,0.29,0.30,0.31]
+		},
+		{
+			"DEG_S" : 20,
+			"MONTH_VALS" : [0.30,0.29,0.28,0.26,0.25,0.25,0.25,0.26,0.27,0.28,0.29,0.30]
+		},
+		{
+			"DEG_S" : 15,
+			"MONTH_VALS" : [0.29,0.28,0.28,0.27,0.26,0.25,0.26,0.26,0.27,0.28,0.29,0.29]
+		},
+		{
+			"DEG_S" : 10,
+			"MONTH_VALS" : [0.29,0.28,0.28,0.27,0.26,0.26,0.26,0.27,0.27,0.28,0.28,0.29]
+		},
+		{
+			"DEG_S" : 5,
+			"MONTH_VALS" : [0.28,0.28,0.28,0.27,0.27,0.27,0.27,0.27,0.27,0.28,0.28,0.28]
+		},
+		{
+			"DEG_S" : 0,
+			"MONTH_VALS" : [0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27,0.27]
+		}
+	]
+}
```

Comment:
Could you put a link to were you got these values from? I trust you got them right, I just think it would be useful to have a reference if we ever need to build upon/explain these numbers...

Reply:
No plagiarism at its best hahahaha... but yea jk I can throw them in there